pkg/services/featuremgmt/toggles_gen_test.go

```diff
@@ -126,7 +126,7 @@ func TestFeatureToggleFiles(t *testing.T) {
 						item.Annotations[utils.AnnoKeyUpdatedTimestamp] = created.String()
 						item.Spec = v // the current value
 					}
-				} else {
+				} else if item.DeletionTimestamp == nil {
```

Comment:
previously this was updating toggles_gen for any old toggle, and then cascading merge issues